Fix a misplaced paren

Seniru opened the pull request:
This is related with and closes #2

Seniru committed:
Fixed a misplaced paren

Seniru committed:
fix: Bug fixes

- Fix a misplaced bracket which caused in malfunctions
- Add an "end" method to proporly ACK a slash interaction
- Minor fixes for some event emittor stuff

Seniru committed:
Merge branch 'main' of https://github.com/Seniru/discordslashcommands into main

Seniru committed:
test: a

mactul reviewed (commented):
I reread the discord.py code to check and you are right, the @client.event decorator allows you to call functions from the client class, it's cleaner
For the end function, returning a callback didn't exist when I wrote the library, I had a quick look and I can see what it does.
However, I don't understand in your code when the end function is called?

If not, yes, I'll add you in the credits, when I'll get the code straight ^^

Seniru commented:
Oh I added your module as a git submodule to my repo and it looks like some extra changes have been commited to this branch I submitted the PR ^^

And the `end` function was actually an extra thing I added as I get `The interaction failed` error otherwise. Also it's intended to be called in user's code, something like this

```py
@client.event
async def on_interaction(*args, **kwargs):
    # ...
    await interaction.end(content = "Handled the interaciton!")
```

Anyways, should I just remove those extra things as this is just supposed to fix a paren or leave it as it is according to your needs
